Lib/fontTools/varLib/instancer/featureVars.py

```diff
@@ -0,0 +1,178 @@
+
+from fontTools.ttLib.tables import otTables as ot
+from fontTools.varLib.models import normalizeValue
+from copy import deepcopy
+import logging
+
+
+log = logging.getLogger("fontTools.varLib.instancer.featureVars")
```

Comment:
we probably don't need this level of granularity and it's fine to use a logger named "fontTools.varLib.instancer" like the parent module